lib/galaxy/datatypes/converters/tabular_to_dbnsfp.xml

```diff
@@ -12,8 +12,8 @@
     </outputs>
     <tests>
         <test>
-            <param name="input" format="tabular" value="gtf_filter_by_attribute_values_list_in3.tabular"/>
-            <output name="dbnsfp" format="snpsiftdbnsfp" value="gtf_filter_by_attribute_values_list_in3.snpsiftdbnsfp"/>
```

Comment:
Should gtf_filter_by_attribute_values_list_in3.snpsiftdbnsfp be removed from https://github.com/galaxyproject/galaxy-test-data ?

Reply:
yes

Reply:
Done in https://github.com/galaxyproject/galaxy-test-data/commit/f82fbd3566dce70e8b1e8a8f43f7c910fb065d10